02_conan_install.sh

```diff
@@ -1,3 +1,3 @@
 #/bin/sh
 conan --version
-conan install conanfile.txt --profile:host=m1 --profile:build=default
+conan install conanfile.txt --profile:host=conan/profiles/m1 --profile:build=conan/profiles/default --generator virtualenv
```

Comment:
We need to use `virtualenv` generator to create the files with all the environment that the build-requires are contributing.